cmd/update/update.go

```diff
@@ -19,6 +19,10 @@ import (
 	"github.com/buildsafedev/bsf/pkg/update"
 )
 
+var updateCmdOptions struct {
+	check bool
```

Comment:
this can be a global variable without a struct

Reply:
I did this as in future many options can be implemented such as updating to a specific package version, etc

Reply:
This entire `cmd/update` package is scoped to have flags for the CLI.  Typically, cobra application flags have them set globally, so it's preferable if we didn't use the struct